### DISCUSSION TITLE
Provide a generator for Java

### DISCUSSION THREAD

**Discussion by roamingthings** [awslabs/smithy#2448] [Ideas]
I like the concept and robustness of Smithy. I also like the code that is generated by the existing generators. In contrast my experience with OpenAPI generators especially for Java has been not very positive.

I would be very happy if you provide a generator for Java.

**Comment by cmoher** [comment]
I have good news!  We are working on one.  We are working diligently to open-source a new smithy-java generator in the coming months.  Stay tuned.

**Comment by cmoher** [comment]
smithy-java is now publicly available in preview! https://github.com/smithy-lang/smithy-java
